server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/UploadStepdefs.java

```diff
@@ -143,7 +143,7 @@ public void userUploadTooBigContent(String username) throws Throwable {
 
     @When("^\"([^\"]*)\" checks for the availability of the upload endpoint$")
     public void optionUpload(String username) throws Throwable {
-        AccessToken accessToken = userStepdefs.getTokenForUser(username);
+        userStepdefs.getTokenForUser(username);
```

Comment:
Remove this line

Reply:
no this is needed to authenticate username, as a difference from the other scenario "optionUploadWithoutAuthentification"

Reply:
Ok, this may help us:
https://github.com/linagora/james-project/pull/1064/commits/a289e4466c8037598657c629243e2cf92228a9c9